daily-challenges/2024-11/05/Ducnq.k11.spec.ts

```
@@ -0,0 +1,85 @@
+//     return result;
+// }
+// console.log(calculateTreasure(["1...5", "2...6", "10...12"])); 
+
+
+function calculateTreasure(sequences: string[]): number[] {
+    let result: number[] = [];
+    
+    for (let element of sequences) {
+        // Kiểm tra định dạng của chuỗi "a...b"
+        const regex = /^(\d+)\.\.\.(\d+)$/;  // Biểu thức chính quy kiểm tra định dạng "a...b"
+        const match = element.match(regex);  // Tìm phần tử phù hợp với regex
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)